api/base/serializers.py

```diff
@@ -1358,20 +1358,18 @@ def to_representation(self, obj, envelope='data'):
             context_envelope = None
         enable_esi = self.context.get('enable_esi', False)
         is_anonymous = is_anonymized(self.context['request'])
-        to_be_removed = set()
-        if is_anonymous and hasattr(self, 'non_anonymized_fields'):
-            # Drop any fields that are not specified in the `non_anonymized_fields` variable.
-            allowed = set(self.non_anonymized_fields)
-            existing = set(self.fields.keys())
-            to_be_removed = existing - allowed
+        not_allowed = []
+        if is_anonymous and hasattr(self, 'anonymized_fields'):
+            # Drop any fields that are specified in the `anonymized_fields` variable.
+            not_allowed = self.anonymized_fields
```

Comment:
Instead of specifying `non_anonymized_fields` on the serializers, thought we should specify `anonymized_fields`.  Serializers with `non_anonymized_fields` specified would require edits to that list nearly every time a field was added.  (This is how we got into the issue where barely any fields were being returned with a registration w/ VOL's.) Cons are that new fields are present by default with VOL's, so we need to be mindful if they need to be anonymized.